src/main/java/net/minecraftforge/event/entity/player/CriticalHitEvent.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+package net.minecraftforge.event.entity.player;
+
+import net.minecraftforge.fml.common.eventhandler.Event.HasResult;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.player.EntityPlayer;
+
+/**
+ * CriticalHitEvent.Decision is fired to determinate is a player should make a critical hit.<br>
+ * This event is fired whenever a player attacks an Entity in
+ * EntityPlayer#attackTargetEntityWithCurrentItem(Entity).<br>
+ * <br>
+ * This event is not {@link Cancelable}.<br>
+ * <br>
+ * This event has a result. {@link HasResult}<br>
+ * DEFAULT: means the vanilla logic will determinate if this a critical hit.<br>
+ * DENY: it will not be a critical hit but the player still will attack<br>
+ * ALLOW: this attack is forced to be critical
+ * <br>
+ * This event is fired on the {@link MinecraftForge#EVENT_BUS}.
+ **/
+@HasResult
+public class CriticalHitEvent extends PlayerEvent
+{
+    private float damageModifier;
+    private final Entity target;
+    
+    public CriticalHitEvent(EntityPlayer player, Entity target, float vanillaDefault)
+    {
+        super(player);
+        this.target = target;
+        damageModifier = vanillaDefault;
+    }
+    
+    /**
+    * The attacked Entity by the Player
+    */
```

Comment:
All three of these javadocs are indented differently. See other events for proper formatting